### PR TITLE
Expose wire instructions for payments/marketplace API

### PR DESCRIPTION
**Pull request by alandn-circle**
A new endpoint is added for acquiring wire instructions for a bank account (`GET /wires/{id}/instructions`).

### CODE REVIEW

**Comment by alandn-circle** [comment]
Not sure how else I should name my file in `pages` - considered just using `instructions.vue`. Is there a better way to name this endpoint re: `pages/debug/...`?

File: lib/wiresApi.ts [hunk 1]
```diff
@@ -94,8 +94,19 @@ function getWireAccountById(accountId: string) {
...
 
+/**
+ * Get Wire Account Instructions
+ * @param {String} accountId
+ */
+function getWireAccountInstructions(accountId: string) {
+  const url = `/v1/banks/wires/${accountId}/instructions`
+
+  return instance.get(url)
+}
```
Comment (timmy-circle) [inline lib/wiresApi.ts:105]: `id` will be enough I believe. 
I think adding the account will cause some confusion cuz we have an account notion for wallet.

**Comment by timmy-circle** [comment]
A minor requst: Would you mind re-order 
Screen Shot 2020-10-06 at 11 37 41 AM

to be 

POST /wire 
GET /wire/:id
GET /wire/:id/instruction

I think that flow would make more sense.

**Comment by timmy-circle** [comment]
I also verified that it works locally against smokebox. So once you resolve my comment, LGTM.

**Comment by timmy-circle** [comment]
> Not sure how else I should name my file in `pages` - considered just using `instructions.vue`. Is there a better way to name this endpoint re: `pages/debug/...`?

I think it's fine as is.

**Comment by kristinfritsch** [comment]
Agree with what Timmy said, otherwise looks good!

**Comment by kristinfritsch** [comment]
I rekicked the PR builder

**Review by kristinfritsch — APPROVED** [review]
👍

**Pull request merged**